src/mutualinfo.jl

```diff
@@ -0,0 +1,40 @@
+# Mutual Information
+
+function _mutualinfo(A::AbstractMatrix{<:Integer})
+    N = sum(A)
+    (N == 0.0) && return 0.0
+
+    aᵢ = sum(A, dims=2)
+    bⱼ = sum(A, dims=1)
+    entA = entropy(A)
+    entArows = entropy(aᵢ)
+    entAcols = entropy(bⱼ)
+
+    hck = (entA - entAcols)/N
+    hc = entArows/N + log(N)
+    hk = entAcols/N + log(N)
+
+    mi = hc - hck
+    nmi = 2*mi/(hc+hk)
+
+    return mi, nmi
+end
+
+"""
+    mutualinfo(a, b) -> Float64
```

Comment:
Maybe we can add a keyword specifying whether we need the normalized MI?
```suggestion
    mutualinfo(a, b; normed::Bool = false) -> Float64
```

`multualinfo(a, b, normed=true)` is better than `mutualinfo(a, b)[2]`